### DISCUSSION TITLE
Upload proof of your academic status

### DISCUSSION THREAD

**Discussion by Steve17250** [community/community#38575] [GitHub Education]
Hi, (before start, sorry for my english I'm french student)

When I submit a document to prove that I am enrolled in this school it does not work! Can you help me? Thank you in advance

**Comment by subidit** [comment]
It usually takes a few days to get approved.
I hope the documents you provided meets their [requirements](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved).

Good luck!

**Comment by alexstrugacz** [comment]
**Hey Steve17250 -** 

My education status request got approved within a few minutes - I believe it is done by a robot. 
I faced a similar problem when trying to apply. I was submitting documents but they were not being accepted. 

First make sure you don't fall under these 3: 

1. You're enrolled in an informal learning program that is not part of the [GitHub Campus Program](https://education.github.com/schools) and not enrolled in a degree or diploma granting course of study.
2. You're pursuing a degree which will be terminated in the current academic session.
3. You're under 13 years old.

Then, you need to make sure that your document has a **date** on it! This date will let GitHub know that you are CURRENTLY enrolled in a school/program. That was what resolved my issue - submit document(s) with dates on them and they will accept it!

Let me know if you face any more issues, Good Luck! @Steve17250

**Comment by samus-aran** [comment]
Hi @Steve17250  👋 , we're happy you're here! You're more likely to get a useful response if you are explicit about what your reasons rejection are (a screenshot may help) and any additional info you think is relevant. Giving a few more details might help someone give you a nudge in the right direction. 👍
